mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientidentifiers/ClientIdentifiersFragment.java

```diff
@@ -130,16 +132,34 @@ public void showUserInterface() {
     public void showClientIdentifiers(List<Identifier> identifiers) {
         this.identifiers = identifiers;
         identifierListAdapter.setIdentifiers(identifiers);
+        identifierListAdapter.notifyDataSetChanged();
```

Comment:
This is a bad way of handling changes on a recycler view. I'd recommend reading Google's Documentation about RecyclerView. 

Hint : You should almost never call ```notifyDataSetChanged()``` method on a RecyclerView's Adapter. There are other methods available for addition, modification, removal etc. 

Also read about ```DiffUtils``` and try to use that here. 

I am aware we use this bad approach all across the app but this is the time you learn and do it and set a precedence for the future devs so we can just point them to your code and they can learn from it.